recipes-wpe/wpeframework/wpeframework_git.bb

```diff
@@ -96,13 +92,26 @@ WPEFRAMEWORK_EXTERN_EVENTS ?= " \
     Location Internet \
 "
 
+def getlayerrevision(d):
+    topdir = bb.data.getVar('TOPDIR', d, True)
+
+    layers = (bb.data.getVar("BBLAYERS", d, True) or "").split()
+    for layer in layers:
+        my_layer = layer.split('/')[-1]
+        if my_layer == 'meta-wpe':
+            return base_get_metadata_git_revision(layer, None)
+
+    return "unknown"
+
+WPE_LAYER_REV = "${@getlayerrevision(d)}"
```

Comment:
This gets the meta-wpe revision to be added to the build config?

Reply:
Yes, so we can see in a build which revision it was build from.

Reply:
Yes, its updating the meta-wpe revision to the TREE_REFERENCE and we can view it from deviceinfo

Reply:
nice!